src/msw/textctrl.cpp

```diff
@@ -829,6 +835,39 @@ WXDWORD wxTextCtrl::MSWGetStyle(long style, WXDWORD *exstyle) const
     return msStyle;
 }
 
+#if wxUSE_RICHEDIT && wxUSE_SPELLCHECK
+
+bool wxTextCtrl::EnableSpellCheck(const bool enable)
+{
+    wxCHECK_MSG((m_windowStyle & wxTE_RICH2), false,
+            "Unable to enable spell check on control "
+            "which does not have wxTE_RICH2 style");
+
+    LPARAM editStyle = SES_USECTF | SES_CTFALLOWEMBED
```

Comment:
Looks like we need `SES_XXX` definitions for MinGW (not -w64 variety) too, could you please add them above together with `IMF_SPELLCHECKING `?

Reply:
Yes, I have been looking at doing this. I was thinking that Appveyor must have a pretty old version of MinGW as they have been there since 2014, but that is MinGW -w64 as you say. 

I will add them alongside IMF_SPELLCHECKING - wasn't sure if this was the best place or in missing.h.

Reply:
I guess we're going to finally drop support for classic MinGW after 3.2.0, together with the other old compilers, but not yet...

As for where to define it, the heuristic is that if it's likely to be needed by multiple files, then it should go into `missing.h`, otherwise be defined just in the only file using it. Here it seems unlikely that we should need them elsewhere, so I'd keep things local.

TIA!